src/untangled/client/core.cljc

```diff
@@ -283,6 +283,65 @@
     {:merge-query merge-query
      :merge-data  merge-data}))
 
+(defn- is-atom?
+  "Returns TRUE when x is an atom."
+  [x]
+  (instance? #?(:cljs cljs.core.Atom
+                :clj  clojure.lang.Atom) x))
+
+(defn- generic-integrate-ident
+  [state ident & named-parameters]
```

Comment:
Since this is a private function, you could have taken the instructions as a sequence instead of varargs. Then you would not need apply later.